src/mono/mono/metadata/custom-attrs.c

```diff
@@ -1807,7 +1807,7 @@ mono_custom_attrs_from_assembly_checked (MonoAssembly *assembly, gboolean ignore
 	return mono_custom_attrs_from_index_checked (assembly->image, idx, ignore_missing, error);
 }
 
-static MonoCustomAttrInfo*
+MonoCustomAttrInfo*
 mono_custom_attrs_from_module (MonoImage *image, MonoError *error)
```

Comment:
Probably don't need to expose this since I don't think you need it.  (See other comment)

Reply:
If I can get a MonoAssembly from a MonoMethod, no problem